### ISSUE TITLE
"Quiet For X hours Y minutes" line is too large

### ISSUE THREAD

**Issue by masutomo** [chriscareycode/nagiostv-react#28]
Hi Chris I like your nagios TV a lot about I found this problem in the 0.5.8, that this Quiet For bar takes up lots for blank space vertically in my Chromium Linux browser. I found this can be fixed by edit the main.XXX.css at:
`QuietFor.quietfor-xlarge-size{padding-top:100px;padding-bottom:100px}`
change to:
`QuietFor.quietfor-xlarge-size{padding-top:10px;padding-bottom:10px}`
Can this be fixed in the next version? Thanks.

**Comment by chriscareycode** [comment]
I did this change on purpose to help signify a longer quiet time vs a short quiet time

Previously I had been using color only to signify how long the quiet time was. red, orange, yellow, green . With red being short quiet time, up to green meaning the quiet time was very long.

In a recent version I added the padding to give a better visual indication of how long the quiet time was with short quiet time being vertically small and long quiet time being vertically long.

The xlarge size is a bit extreme with 100 pixels above and below. I will consider to make this less extreme for future version, or to add an option like "Show Quiet time length visually" to turn it on.
